frontend/src/pages/GraphPF/components/node.tsx

```diff
@@ -72,6 +73,10 @@ type BaseNodeProps = {
   element: Node;
   droppable?: boolean;
   hover?: boolean;
+  isFocused?: boolean;
+  isFind?: boolean;
+  hasSpans?: boolean;
+  isUnhighlighted?: boolean;
```

Comment:
Let's move our additions to the bottom, under a short comment.  We can alphabetize our section :-)